### ISSUE TITLE
Brownie Verification Error (no idea how to debug)

### ISSUE THREAD

**Issue by FBosler** [eth-brownie/brownie#1345]
### Environment information

- `brownie` Version: 1.17.0
- `ganache-cli` Version: 6.12.2
- `py-solc-ast` Version: 1.2.9
- `py-solc-x` Version:1.1.1

* Python Version: 3.9.7
* OS: macOs BIG Sur 11.6

### What was wrong?

Getting an error when verifying an otherwise functional contract. Having an insanely hard time debugging verification errors. Check repo for exact reproduction

- Repo: https://github.com/FBosler/brownie_verification_error

### How can it be fixed?

No idea

**Comment by skellet0r** [comment]
![DeepinScreenshot_Alacritty_20211124070031](https://user-images.githubusercontent.com/23556333/143190275-dc3bf1c5-142e-4f32-85c8-7c253a73f3ad.png)

I used brownie v1.17.1

**Comment by FBosler** [comment]
@skellet0r thx for checking, will run after work too! Did you try it with brownie 1.17.0 as well?

**Comment by skellet0r** [comment]
No, but after downgrading to 1.17.0 and looking at the verification info I see it's missing stuff. But yeah latest brownie version seems to have resolved this
